docker-compose/run-parodos-locally.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```

Comment:
I would add a comment in the Readme, but looks good to me :-)

Reply:
Awesome! Thank you